riverdriver/riverdatabasesql/go.mod

```diff
@@ -7,6 +7,7 @@ toolchain go1.23.0
 require (
 	github.com/jackc/pgx/v5 v5.7.1
 	github.com/lib/pq v1.10.9
+	github.com/riverqueue/river v0.12.0
```

Comment:
@brandur hmm, I think I've created a cycle here? Both drivers are now calling `dbunique.UniqueBitmaskToStates()` in `internal/dbunique`, which has made them both take a dependency on the main river package. And the main package depends on the drivers too due to testing:

https://github.com/riverqueue/river/blob/5e3533556d8e6fc77ae11ce36577113055a82434/go.mod#L12-L13